spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-binder-sample/src/main/java/com/example/SinkExample.java

```diff
@@ -35,7 +37,17 @@
 
 	@StreamListener(Sink.INPUT)
 	public void handleMessage(UserMessage userMessage) {
+		if (userMessage.isThrowError()) {
+			throw new RuntimeException("An error was triggered in the message handler!");
+		}
+
 		LOGGER.info("New message received from " + userMessage.getUsername() + ": " + userMessage.getBody() +
 				" at " + userMessage.getCreatedAt());
 	}
+
+	@StreamListener("errorChannel")
+	public void handleErrorMessage(Message<MessagingException> message) {
```

Comment:
How does this `errorChannel` get hooked up into the binder? By convention of the name?
We might need to document it.

Reply:
You don't. This one comes from Spring Integration: https://docs.spring.io/spring-integration/docs/current/reference/html/#channel-special-channels
And it is indeed is wired in Spring Cloud Stream for error handling.

I'm fully don't see reason to document anything in this project. Unless you don't have links to SCSt docs.

I somehow assume when you try to use Pub/Sub Binder, you are already familiar with SCSt core...